src/styles/header/header.css

```diff
@@ -1,5 +1,6 @@
 
 .header {
+    position: relative;

```

Comment:
All elements with `position: absolute` need a continued to base on. This issue can help `position: relative`

Check [the official doc](https://www.w3schools.com/css/css_positioning.asp) for more